deploy/frontend.setup

```diff
@@ -0,0 +1,50 @@
+from setuptools import setup, find_packages
+
+install_requires = [
+    "pyzmq>=15.3.0",
+    "msgpack >= 1.0.0",
+    "Werkzeug >= 1.0.0",
+    "PyYAML>=3.11",
+    "Flask >= 1.1.0",
+    "Flask-Mail >= 0.9.1",
+    "pymongo>=3.2.2",
+    "oauth2>=1.9.0.post1",
+
+    #"docker>=2.5.0",
+    #"docutils>=0.14",
+    "pymongo>=3.2.2",
+    "PyYAML>=3.11",
+    "Jinja2 >= 2.10",
+    "lti>=0.9.0",
+    "oauth2>=1.9.0.post1",
+    "httplib2>=0.9",
+    "watchdog >= 0.8.3",
+    "msgpack >= 1.0.0",
+    "pyzmq >= 15.3.0",
+    "natsort >= 5.0.1",
+    "psutil >= 4.4.2",
+    "zipstream >= 1.1.4",
+    "WsgiDAV >= 3.0.0",
+    "Werkzeug >= 1.0.0",
+    "itsdangerous >= 1.1.0",
+    "Flask >= 1.1.0",
+    "Flask-Mail >= 0.9.1",
+    "importlib_metadata >= 3.7.0",
+    'dataclasses >= 0.8; python_version < "3.7.0"',
+    "pytidylib>=0.2.4",
+    "sphinx-autodoc-typehints>=1.12.0",
+    "sh"
+]
+
+setup(
+    name="INGInious",
+    packages=find_packages(),
+    install_requires=install_requires,
+    include_package_data=True,
+    extras_require={
```

Comment:
Probably not needed

Reply:
Removed in https://github.com/UCL-INGI/INGInious/pull/882/commits/13e381ef3da8a4cc068bcf455fc441f3f4a671da.